### ISSUE TITLE
Logging

### ISSUE THREAD

**Issue by kravchenko-rn** [BotMill/fb-botmill#124]
@alvin-reyes fb-botmill logs everything to console. Can I make it to put logs in a file?
Thanks in advance.

**Comment by alvin-reyes** [comment]
Yes. We've been planning to do this for a while, so yeah it would be an
awesome addition.


 Please create a Pull Request once its done and set me as reviewer.

**Comment by andrefsore** [comment]
I would suggest not to pack logback.xml in the jar file. Let the user decide where and what to log

**Comment by alvin-reyes** [comment]
@andrefsore , do you have any suggestion how can we do it better?

**Comment by andrefsore** [comment]
The developer who imports the lib, should create theirs own logback.xml. I had to create logback-test.xml  in order to override logback.xml that was packed in fb-botmil.jar, because I wanted to set the logging level by myself, for instance.
